deployment/configgen/main.cpp

```diff
@@ -494,12 +494,12 @@ int processRequest(const char* in_cfgname, const char* out_dirname, const char*
     {
       IPropertyTree* pDropZone = &dropZonesInsts->query();
       StringBuffer computerName(pDropZone->queryProp(XML_ATTR_COMPUTER));
-      xPath.clear().appendf("Hardware/Computer[@name=\"%s\"]", computerName.str());
-      IPropertyTree* pComputer = pEnv->queryPropTree(xPath.str());
-      if (pComputer)
-      {
-        const char* netAddr = pComputer->queryProp("@netAddress");
 
+      Owned<IPropertyTreeIterator> serverList = pDropZone->getElements("ServerList");
```

Comment:
We should continue to support legacy style DZ structures,
continue looking for "Hardware/Computer[@name=\"%s\"]"
here as well as ServerList

BTW we could use xPath.setf rather than the legacy xPath.clear().appendf